addons/mail/tools/link_preview.py

```diff
@@ -3,6 +3,7 @@
 
 from lxml import html
 import requests
+from urllib3.exceptions import LocationParseError
```

Comment:
(minor) imports should be alphabetical

Reply:
:socks: +1

Reply:
mm i don't get it, i personally think that they are already ordered (I order them based on library names and not the imported class/function names)

Reply:
Actually I don't know how we are supposed to order them, so I generally split global import from remaining imports. But anyway this has no real interest, better fix the issue :)

Reply:
In JS I order the them by name/path rather than keyword, but the python guideline is rather clear about having `import` after `from`.

I'm not sure what's the point, I guess easier to just select all lines and click order in editor to automatically do it.

https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports